cmd/logviewer/main.go

```diff
@@ -205,57 +228,54 @@ func listeningOnWebSocket() {
 		if msgType == websocket.CloseMessage {
 			return
 		}
-		if err != nil {
-			isConnectionClosed := strings.Contains(err.Error(), "websocket: close")
-			if !isConnectionClosed {
-				log.Error("logviewer websocket error", "error", err.Error())
-				manualStop <- struct{}{}
-
-			}
-			return
+		if err == nil {
+			outputMessage(message)
+			continue
 		}
 
-		outputMessage(message)
+		_, isConnectionClosed := err.(*websocket.CloseError)
+		if !isConnectionClosed {
+			log.Error("logviewer websocket error", "error", err.Error())
+			manualStop <- struct{}{}
+		}
+		return
```

Comment:
What can we do here extra, is instead of return, the connection can be set to nil and as long as it is nil we can try and redial. There could be the situation where we stop the node (for an update) and the logview app can reconnect automatically without having to close it and reopen it

Reply:
Good ideea, implementing...